docs/sql-migration-guide.md

```diff
@@ -226,6 +226,32 @@ license: |
 
   - Since Spark 3.0, when casting string value to date, timestamp and interval values, the leading and trailing white spaces(<= ACSII 32) will be trimmed before casing, e.g. `cast('2019-10-10\t as date)` results the date value `2019-10-10`. In Spark version 2.4 and earlier, only the trailing space will be removed, thus, the result is `null`.
 
+  - Since Spark 3.0, we pad decimal numbers with trailing zeros to the scale of the column for Hive result, for example:
```

Comment:
@wangyum, can you clarify what "Hive result" means?

Reply:
https://github.com/apache/spark/blob/051e691029c456fc2db5f229485d3fb8f5d0e84c/sql/core/src/main/scala/org/apache/spark/sql/execution/HiveResult.scala#L33-L37

Reply:
I meant to clarify it in this migration guide ...

Reply:
How about?
Since Spark 3.0, we pad decimal numbers with trailing zeros to the scale of the column for `spark-sql` interface.

Reply:
+1